### DISCUSSION TITLE
Noob question: Set up for clarity

### DISCUSSION THREAD

**Discussion by Muizzyranking** [HiPhish/rainbow-delimiters.nvim#146] [Q&A]
I have been trying to get this to work with clarity and I there is no breakthrough, but my main question is where do I put the file? i have been putting the file in `$HOME/.local/share/nvim/lazy/rainbow-delimiters.nvim/queries/clarity/rainbow-delimeters.scm`. Another question, say I have the grammer.js file for this language and the highlights.scm for syntax highlighting, what do I look for? how do I know what to target?

**Comment by HiPhish** [comment]
Did you install the parser for Clarity?

**Comment by Muizzyranking** [comment]
yes, I did. It was a custom paser from [https://github.com/xlittlerag/tree-sitter-clarity.git](https://github.com/xlittlerag/tree-sitter-clarity.git) that I added to treesitter nvim and highlights works.

**Comment by HiPhish** [comment]
Have you tried putting the query file in `~/.config/nvim/after/queries/clarity/rainbow-delimiters.scm`? I don't use Lazy, so I have no idea what kind of magic it does to the runtime path. Can you post your query file here? Maybe there is something wrong with the patterns.
